### ISSUE TITLE
[support]: amrex test failure on JLSE

### ISSUE THREAD

**Issue by koysean** [E4S-Project/e4s#135]
### Name of Software

amrex

### Contact Details

skoyama@anl.gov

### HPC System

JLSE

### Request Description

amrex test fails with errors relocation overflow errors.

### Relevant log output

```shell
AdvancePhiAllLevels-bf5c2c.cpp:(.text+0x1e62): relocation truncated to fit: R_X86_64_PC32 against symbol `amrex::Gpu::Device::gpu_stream'
```

**Comment by eugeneswalker** [comment]
Hi Sean, thank you for reporting this. We will investigate this and post updates here as we discover why this is happening and what is required to resolve.

**Comment by eugeneswalker** [comment]
I was able to reproduce a `spack test` failure for `amrex@23.05` on the JLSE deployment you told me about offline. The failure message I am seeing is different than the one you posted above. We can continue our discussion through CNDA approved channels so we can find out why the error you report is different than what I can reproduce.

In the meantime, I went ahead and ran `spack test` on the updated version `amrex@23.08` to be deployed as part of E4S 23.08 and that test passes. The version of Spack we are using for E4S 23.08 has three more months of development fixes compared to E4S 23.05 where this error originated for you, including some updates to `amrex`'s Spack test code (see https://github.com/spack/spack/pull/35698 for one example).

Let us keep this issue open until we can mark this resolved based on further investigation.

**Comment by eugeneswalker** [comment]
Hi Sean, 

This error is resolved in the latest `amrex +sycl` release included with E4S 23.08 on Sunspot.

Here is output showing `spack test run` from a compute node using the E4S 23.08 release.
```
$> spack test run amrex+sycl
==> Spack test 2tdtmijfnedxda2y6hbgbprkbopm5bb6
==> Testing package amrex-23.08-ftppgw7
============================== 1 passed of 1 spec ==============================

$> spack find -lv /ftppgw
-- linux-sles15-icelake / oneapi@2023.1.0 -----------------------
ftppgw7 amrex@23.08~amrdata~cuda~eb~fortran~hdf5~hypre~ipo+linear_solvers+mpi~openmp~particles~petsc~pic~plotfile_tools~rocm~shared~sundials+sycl~tiny_profile build_system=cmake build_type=Release dimensions=1,2,3 generator=make precision=double
==> 1 installed package
```

Are you able to verify this is resolved for you?

**Comment by eugeneswalker** [comment]
Per our communication offline @koysean I will close this issue as resolved for now. If you discover this to be a problem in the future we will re-open. Thanks!
